react/react1/week1/todo-list/src/TodoList.js

```diff
@@ -0,0 +1,11 @@
+import {TodoItems} from "./TodoItems";
+import React from "react";
+export function TodoList(props) {
+    return (
+        <ul>
```

Comment:
Again, I prefer not to inline the map operations in the JSX. Assign it to a const before the return and give it a meaningful name